butterknife-annotations/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
I'd prefer to have one `.gitignore` in the root only. It has "build" already so this can be deleted.